### ISSUE TITLE
Absorbed diona nymphs have several issues

### ISSUE THREAD

**Issue by IAmBigCoat** [ParadiseSS13/Paradise#5585]
**Problem Description**:
 - Absorbed nymphs don't have the option to leave the gestalt unless they log out and log back in.
 - Absorbed nymphs can bite their host diona

**What did you expect to happen**: 
The opposite of those things.

**What happened instead**:
Those things exactly.

**Why is this bad/What are the consequences:**
Unavoidable murder by absorbed nymph.

**Steps to reproduce the problem**:
Attempt to pet a nymph as diona, wait for them to become enraged with their inability to leave you and subsequently bite you to death.

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was
anything else out of the ordinary happening?)**: 
Probably doesn't work with humans or IPCs.

**Comment by TheDZD** [comment]
Yeah, there are definitely issues with this that have been around for over 2 years, although I think nymphs are supposed to be able to hop out by resisting.

**Comment by TwistedAkai** [comment]
This issue also arises if the nymph willfully merges.

**Comment by TwistedAkai** [comment]
A couple old-ish BYOND bugs that may be relevant to the verbs issue.

http://www.byond.com/forum/?post=2018444
http://www.byond.com/forum/?post=1833666

**Comment by TwistedAkai** [comment]
https://github.com/ParadiseSS13/Paradise/blob/eac5ae96594da588f481cf2c2078da4fc7b276d2/code/modules/mob/living/simple_animal/friendly/diona.dm#L94 Moving this line down to after the verb swap -might- fix it? Assuming not being on a /turf is the problem.

**Comment by KasparoVy** [comment]
IDK if this is still a thing.

**Comment by Spacemanspark** [comment]
It is.

**Comment by Fox-McCloud** [comment]
@KasparoVy 

AI's have the same problems when they gain/lose malf status; basically when a verb is added or removed, it has to be the users default view for it to update (as a hacky way of resolving this, every time a power is purchased, the view resets to the core).

You may be able to do something similar; it's about the only way as it's a BYOND issue.

Alternatively you could axe the verbs altogether and make them innate actions to the mob itself (which would be better, in the long run).
